ephios/core/migrations/0022_identityprovider.py

```diff
@@ -0,0 +1,103 @@
+                    models.CharField(
+                        help_text="Your client id provided by the OIDC provider.",
+                        max_length=255,
+                        verbose_name="client id",
+                    ),
+                ),
+                (
+                    "client_secret",
+                    models.CharField(
+                        help_text="Your client secret provided by the OIDC provider.",
+                        max_length=255,
+                        verbose_name="client secret",
+                    ),
+                ),
+                (
+                    "scopes",
+                    models.CharField(
+                        default="openid profile email",
+                        help_text="The OIDC scopes to request from the provider. Separate multiple scopes with spaces. Use the default value if you are unsure.",
+                        max_length=255,
+                        verbose_name="scopes",
+                    ),
+                ),
+                (
+                    "authorization_endpoint",
+                    models.URLField(
+                        help_text="The OIDC authorization endpoint.",
+                        verbose_name="authorization endpoint",
+                    ),
+                ),
+                (
+                    "token_endpoint",
+                    models.URLField(
+                        help_text="The OIDC token endpoint.", verbose_name="token endpoint"
+                    ),
+                ),
+                (
+                    "userinfo_endpoint",
+                    models.URLField(
+                        help_text="The OIDC user endpoint.", verbose_name="user endpoint"
+                    ),
+                ),
+                (
+                    "end_session_endpoint",
+                    models.URLField(
+                        blank=True,
+                        help_text="The OIDC end session endpoint, if supported by your provider.",
+                        null=True,
+                        verbose_name="end session endpoint",
+                    ),
+                ),
+                (
+                    "jwks_uri",
+                    models.URLField(
+                        blank=True,
+                        help_text="The OIDC JWKS endpoint. A less secure signing method will be used if this is not provided.",
+                        null=True,
+                        verbose_name="JWKS endpoint",
+                    ),
+                ),
+                (
+                    "default_groups",
+                    models.ManyToManyField(
+                        blank=True,
+                        help_text="The groups that users logging in with this provider will be added to.",
+                        to="auth.group",
+                        verbose_name="default groups",
+                    ),
+                ),
+            ],
+        ),
+    ]
```

Comment:
We should add an operation migrating the old env-based settings to a new identity provider automatically?!